permchain/pregel/reserved.py

```diff
@@ -0,0 +1,8 @@
+from enum import StrEnum
+
+
+class ReservedChannels(StrEnum):
+    """Channels managed by the framework."""
+
+    is_last_step = "is_last_step"
```

Comment:
Is step the correct way of calling this? Is step referring to an update step rather than agent step here?

Reply:
A Pregel algorithm step